### PR TITLE
[battery] small cleanup

### PR DESCRIPTION
**Pull request by mzieg**
Fixing a small inelegance (batteryStateRaw gets called 3x more than it needed too, although timestamp-check would have de-bounced anyway).

### CODE REVIEW

File: WasatchNET/EEPROM.cs [hunk 1]
```diff
@@ -1338,7 +1338,7 @@ public bool read()
                     baudRate = 0;
 
                     hasCooling = false;
-                    hasBattery = true;
+                    hasBattery = true;   // MZ: why?
```
Comment (mzieg) [inline WasatchNET/EEPROM.cs:1341]: What was the reasoning here?

File: WasatchNET/Spectrometer.cs [hunk 2]
```diff
@@ -514,24 +514,25 @@ public uint batteryStateRaw
                 if (!eeprom.hasBattery)
                     return 0;
 
+                // don't check the battery faster than 1Hz
                 DateTime now = DateTime.Now;
-                DateTime nextCheck = batteryStateTimestamp_.AddSeconds(1);
-                if (batteryStateRaw_ != 0 && now < nextCheck)
```
Comment (mzieg) [inline WasatchNET/Spectrometer.cs:519]: I didn't see the need for this zero check.

File: WasatchNET/Spectrometer.cs [hunk 3]
```diff
@@ -1926,6 +1927,7 @@ internal byte[] getCmd2(Opcodes opcode, int len, ushort wIndex = 0, int fakeBuff
                 {
                     logger.error("getCmd2: failed to get SECOND_TIER_COMMAND {0} (0x{1:x4}) via DEVICE_TO_HOST ({2} of {3} bytes read, expected {4} got {5})",
                         opcode.ToString(), cmd[opcode], bytesRead, len, expectedSuccessResult, result);
+                    logger.hexdump(buf, $"{opcode} result");
```
Comment (mzieg) [inline WasatchNET/Spectrometer.cs:1930]: maybe this will tell us something someday

File: WasatchNET/EEPROM.cs (hunk 1 above)
Comment (trestoh) [reply WasatchNET/EEPROM.cs:1341]: I believe I was told at some point these had batteries...I really can't remember, probably can switch it

**Comment by mzieg** [comment]
merged

**Pull request merged**